### PR TITLE
feat: use playground themes

### PR DESCRIPTION
**Pull request by harish-sethuraman**
#### Prerequisites checklist

-   [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What changes did you make? (Give an overview)

added code mirror theme from playground

#### Related Issues

fixes https://github.com/eslint/code-explorer/issues/62

#### Is there anything you'd like reviewers to focus on?
- Do we keep all the color variables as such or do we remove unused variables? 
- Fixed selection color, bracket color and dark mode
- Report bugs if any

### CODE REVIEW

**Review by nzakas — APPROVED** [review]
:fire: Looks great! Would like @amareshsm to approve before merging.

**Comment by amareshsm** [comment]
Still missing the arrow in the GitHub button

we can use [arrow-up-right](https://lucide.dev/icons/arrow-up-right) from lucide icons. 

src/components/cta.tsx
```
import { ArrowUpRight } from "lucide-react";
import { Button } from "./ui/button";
import type { FC } from "react";

export const CallToAction: FC = () => (
	<div className="light">
		<Button asChild size="sm">
			<a
				href="https://github.com/eslint/code-explorer"
				target="_blank"
				rel="noreferrer"
			>
				<span className="hidden sm:block">GitHub</span>
				<span className="sm:pl-1">
					<ArrowUpRight size={16} />
				</span>
			</a>
		</Button>
	</div>
);

```

File: src/components/cta.tsx [hunk 1]
```diff
@@ -11,8 +11,8 @@ export const CallToAction: FC = () => (
 				rel="noreferrer"
 			>
 				<span className="hidden sm:block">GitHub</span>
-				<span className="sm:hidden">
-					<ExternalLinkIcon size={16} />
+				<span className="sm:ml-2">
+					<ExternalLink size={16} />
```
Comment (amareshsm) [inline src/components/cta.tsx:15]:
```suggestion
					<ArrowUpRight size={16} />
```
The icon does not align with the mockups.

File: src/components/cta.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { ExternalLinkIcon } from "lucide-react";
+import { ExternalLink } from "lucide-react";
```
Comment (amareshsm) [inline src/components/cta.tsx:1]:
```suggestion
import { ArrowUpRight } from "lucide-react";
```

File: src/components/cta.tsx (hunk 1 above)
Comment (harish-sethuraman) [reply src/components/cta.tsx:15]: Its fine the external link looks better than the arrow icon.

File: src/components/cta.tsx [hunk 3]
```diff
@@ -11,8 +11,8 @@ export const CallToAction: FC = () => (
 				rel="noreferrer"
 			>
 				<span className="hidden sm:block">GitHub</span>
-				<span className="sm:hidden">
-					<ExternalLinkIcon size={16} />
+				<span className="sm:ml-2">
```
Comment (amareshsm) [inline src/components/cta.tsx:14]:
```suggestion
				<span className="sm:ml-1">
```
`ml-2`

image

`ml-1`

image



`ml-1` is enough I guess.

Comment (harish-sethuraman) [reply src/components/cta.tsx:14]: shadcn UI had ml-2/mr-2 for buttons with icons and it felt right thats why maintained it

**Comment by amareshsm** [comment]
The text disappears when I click on any word in the editor.
https://github.com/user-attachments/assets/2135fdd5-5432-4149-bc78-588cf2f89be9

**Comment by harish-sethuraman** [comment]
Nice catch! this should be fixed now. Please report if any will fix them.

**Review by amareshsm — APPROVED** [review]
LGTM

**Pull request merged**